torch/csrc/jit/codegen/cuda/python_frontend/python_bindings.cpp

```diff
@@ -318,23 +324,25 @@ void initNvFuserPythonBindings(PyObject* module) {
       py::return_value_policy::reference);
 
   NVFUSER_PYTHON_BINDING_BINARY_OP("add", add)
-  NVFUSER_PYTHON_BINDING_BINARY_OP("and_op", andOp)
```

Comment:
I changed this part to have the same order as `arith.h`, so that it would be easier to compare the list one by one.